Model/Source/AbstractInsurancePossibilities.php

```diff
@@ -4,11 +4,27 @@
 
 namespace MyParcelNL\Magento\Model\Source;
 
+use Exception;
 use Magento\Framework\Data\OptionSourceInterface;
+use MyParcelNL\Sdk\src\Factory\ConsignmentFactory;
+use MyParcelNL\Sdk\src\Model\Consignment\AbstractConsignment;
+use MyParcelNL\Sdk\src\Services\CountryCodes;
 
 abstract class AbstractInsurancePossibilities implements OptionSourceInterface
```

Comment:
Deze hele class kan direct in CarrierInsurancePossibilities, er zijn geen andere classes die dit extenden

Reply:
De class is verplaatst zoals gevraagd